bin/find-orphaned-base

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
+
+#
+# This script checks the ownCloud documentation manual(s) to 
+# see if attachment/example/image files are orphaned.
+# The file is included by another script which defines the variables needed !
+#
+# Note that filenames MUST NOT contain blanks
+
+# check if directly called and exit in case
+	if (( ! ${#AVAILABLE_MANUALS[@]} )); then
+		echo -e "\e[1;31m This script must be called via another script defining the environment.\n Exiting\e[1;37m"
+		exit
+	fi
+
+function start_scan()
+{
+	local list
+	local manual_postfix=
+	local found_find=()
+	local found_fixed=()
+	local found_grep=()
+	local grep_list=
+	local not_linked=()
+	local count_not_linked=
+	local is_image=
+
+	list=$(echo ${AVAILABLE_MANUALS[@]} | sed -r 's/[ ]+/, /g')
+
+	echo
+	echo "Checking for orphaned ${TASK} files in manual: ${list}."
+	echo "Please run this script from your docs root"
+	echo
+
+	for manual in "${AVAILABLE_MANUALS[@]}"; do
+
+		unset found_find
+		unset found_fixed
+		unset found_grep
+
+		# the base directory name depends if there is a single or are multiple manuals available
+		if [[ ${manual} = ${ROOT_NAME} ]]; then
+			BASE_DIR="./modules/${manual}"
+		else
+			BASE_DIR="./modules/${manual}${MANUAL_POSTFIX}"
+		fi
+
+		#echo "${BASE_DIR}"
+		#echo "${BASE_DIR}/${TARGET_DIR}"
+		#echo "${BASE_DIR}/pages"
+		#exit
+
+		[[ ! -e "${BASE_DIR}" ]] && echo "Cannot find ${manual} manual." && continue
+
+		echo -e "\e[1;32mChecking the ${manual} manual \e[0m"
+
+		# return all files found, but omit an error message if TARGET_DIR is not present 
```

Comment:
```suggestion
		# return all files found, but emit an error message if TARGET_DIR is not present 
```

Reply:
omit is correct, do not return an error message if target_dir is not found